src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/OverhangFixingManager.java

```diff
@@ -50,7 +65,8 @@
 
     // the read cache
     private static final int initialCapacity = 5000;
```

Comment:
The other values MAX_RECORDS_IN_MEMORY, etc, should probably not be uppercase since they're not really constants in the absolute sense of the word, they're just final fields.